src/menu.js

```diff
@@ -61,7 +61,7 @@ function menuTempl() {
       },
       {
         label: 'Close',
-        accelerator: 'CommandOrControl+W',
+        accelerator: 'CommandOrControl+Q',
```

Comment:
Just tested on Windows: `Alt+F4` already works.
In fact, the previous key mapping, `Ctrl-W`, also worked on all platforms. It was just `W` instead of `Q`, and was therefore less intuitive.